### ISSUE TITLE
Re: 2016-11-04 issue: negative grid indexes (between 1/-1) do not necessarily span all columns

### ISSUE THREAD

**Issue by emilbjorklund** [simevidas/webplatformdaily-site#108]
Small but important detail: negative indexes in grid line placement only apply to the explicitly defined grid. It's something that bit me in my early research of grid, and potentially an annoyance for newcomers. Maybe change the wording to say `span all explicitly defined columns`?

(I've [commented on the CSS Tricks article](https://css-tricks.com/things-ive-learned-css-grid-layout/#comment-1604838) too, with an explanation of why this is and a demo)

**Comment by simevidas** [comment]
In your demo, “this will only span one column” should say “this will only span one row”, I think.

Spanning all rows is a different use case (I kind-of view everything through use cases 😊); this one’s about spanning all *columns* in an “adaptive” grid. So, if we focus on the horizontal axis, could you give an example of a scenario where `1/-1` would not work?

I’ve found one, I think:

```css
grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
```
Source: [Rachel’s article](https://www.rachelandrew.co.uk/archives/2016/04/12/flexible-sized-grids-with-auto-fill-and-minmax/).

So, this creates a variable amount of columns, depending on the grid width, and from what I can see, the `1/-1` value does not work.

**Comment by emilbjorklund** [comment]
@simevidas Yes - that is exactly the type of scenario I mean! The placement algorithm generally works exactly the same regardless of direction, so any case where there is not a set number of columns or rows can trip people up if they expect a grid item to span using negative indexes.

Good catch on the typo!
